certified-connectors/TPS & CTPS Checker/Readme.md

```diff
@@ -0,0 +1,20 @@
+# TPS & CTPS Checker
+TPS & CTPS Checker allows you to query the UK Telephone Preference Service for a given UK Telephone Number.
```

Comment:
please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). All of the required headings should be present.